docs/en/userguide/index.md

```diff
@@ -0,0 +1,23 @@
+title: Share draft content
+summary: Share draft page content with non-CMS users.
+
+# Share draft content
+
+The [Share draft content](https://github.com/silverstripe/silverstripe-sharedraftcontent/) module provides the ability for SilverStripe users to share draft page content with non-CMS users.
+
+This enables Content authors to generate a private link which can be shared securely with anyone. Each preview link is valid for 30 days.
```

Comment:
Why the capitalisation on Content? It also enables it for other roles, perhaps we could say "This enables CMS users to..." since we're referring to sharing it with _non_ CMS users above

Reply:
Caps lock was a copy and paste fail. Sure thing, will change now.